### ISSUE TITLE
A Clerical Error in the README.md

### ISSUE THREAD

**Issue by DiceyWang** [graknlabs/kglib#80]
Maybe there is a clerical error that exists in the sentence :"Delete all appendix attributes from both **animaltrade_train** and animaltrade_test keyspaces. This is the label we will predict in this example, so it should not be present in Grakn otherwise the network can cheat"

Here the **animaltrade_train** may should be **animaltrade_eval**.  Just let me know if that's right.

**Comment by jmsfltchr** [comment]
Hi, so in fact the script has been written to be generic, and accept an evaluation keyspace, but in this case the data is only partitioned into `animaltrade_train` and `animaltrade_test`, so it is correct as it stands. We definitely want to remove the stored ground truth labels from the test set as well as the training set.
